features/movebefore.yml

```
@@ -0,0 +1,8 @@
+name: moveBefore()
+description: The `moveBefore()` DOM methods relocate a node while preserving its state. For example, you can move an element without disrupting focus or move an iframe without reloading it.
```

I think this is the list of states that get preserved:

- `<iframe>` elements remain loaded.
- The active element retains focus.
- Popovers and dialogs remain open.
- CSS transitions and animations continue to run.

Maybe mention one more:

```suggestion
description: The `moveBefore()` DOM method relocates a node while preserving its state. For example, you can move the active element without losing focus, move an animated element without reseting the animation, or move an iframe without reloading its content.
```